src/Compilers/VisualBasic/Portable/Parser/Parser.vb

```diff
@@ -28,7 +28,7 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.Syntax.InternalSyntax
         Private _possibleFirstStatementOnLine As PossibleFirstStatementKind = PossibleFirstStatementKind.Yes
         Private _recursionDepth As Integer
         Private _evaluatingConditionCompilationExpression As Boolean
-        Private ReadOnly _scanner As Scanner
+        Friend ReadOnly _scanner As Scanner
```

Comment:
`_scanner` should not be exposed. `_scanner.Options` can be exposed as an `Options` property instead.